src/Plugin/video_embed_field/Provider/GoogleDrive.php

```diff
@@ -37,7 +37,19 @@ public function renderEmbedCode($width, $height, $autoplay) {
   public function getRemoteThumbnailUrl() {
     global $base_url;
     global $base_path;
-    return $base_url . $base_path . drupal_get_path('module', 'stanford_media') . '/img/google-drive.png';
+    return $base_url . $base_path . $this->getModulePath() . '/img/google-drive.png';
+  }
+
+  /**
+   * Get the path to this module.
+   *
+   * @return string
+   *   Drupal relative path to module.
+   *
+   * @codeCoverageIgnore
```

Comment:
Do you need a comment why you're ignoring code coverage?

Reply:
i'll add one. But so you dont have to hunt down the comment:
> Ignore code coverage since the global function is not declared during unit tests.

Reply:
for more information: https://www.drupal.org/docs/8/phpunit/unit-testing-more-complicated-drupal-classes